src/hotspot/cpu/ppc/ppc.ad

```diff
@@ -2865,8 +2889,9 @@ encode %{
                                 ra_->get_reg_second(this), ra_->get_reg_first(this));
 
     // Push new nodes.
+    if (loadConLNodes._small)    nodes->push(loadConLNodes._small);
     if (loadConLNodes._large_hi) nodes->push(loadConLNodes._large_hi);
-    if (loadConLNodes._last)     nodes->push(loadConLNodes._last);
+    if (loadConLNodes._large_lo) nodes->push(loadConLNodes._large_lo);
```

Comment:
loadConLNodesTuple_create initializes loadConLNodes._last points to the node that is the same as either loadConLNodes._small or loadConLNodes._larege_lo.  So load_ConLNode is added twice if we don't remove _last checking.  (I actually made this bug and spent a few days to fix it...)

The correct code here should be either: 1) use the same code as that before change, i.e., don't add _small and _large_lo, and only use _last (and _large_hi), or 2) use _small and _large_lo, and remove _last, as I modified.

I chose the option 2 to avoid confusion and to make the change symmetrical to change at [L.3459](https://github.com/openjdk/jdk/pull/4267/commits/403b789cc068ea74a0768406852bf79149b23e32#diff-d21a64a4949f298476bf91083d3b956face9a6393a08a706b071068898533082R3459), where adding _small is mandatory and _last points to other node.

If you (or other reviewers) think the option 1 is better, I can revert this change and add comments as a caveat.

Reply:
I double checked the code and I realized we can't remove _last because the non-"ABI_ELFv2" version of postalloc_expand_java_to_runtime_call() uses _last field in more complex way.  It only uses _last, and set _small, _large_hi, and _large_lo to NULL.  So I think it's better to revert the changes w.r.t. _last and add comment to avoid confusion.